pytorch_lightning/utilities/cloud_io.py

```diff
@@ -35,11 +35,12 @@ def modern_gfile():
     file operations
     """
     tb_version = version.parse(tensorboard.version.VERSION)
-    modern_gfile = tb_version >= version.parse('2.0')
+    modern_gfile = tb_version >= version.parse("2.0")
+    return modern_gfile
 
 
-def cloud_open(path: pathlike, mode: str, newline:str = None):
-    if sys.platform == "win32":
+def cloud_open(path: pathlike, mode: str, newline: str = None):
+    if platform.system() == "Windows":
         log.debug(
             "gfile does not handle newlines correctly on windows so remote files are not"
             "supported falling back to normal local file open."
```

Comment:
```suggestion
            "gfile does not handle newlines correctly on windows so remote files are not"
            " supported falling back to normal local file open."
```
I think this is needed or else there is no space